.github/workflows/node.js.yml

```
@@ -1,31 +1 @@
-
-    steps:
-    - uses: actions/checkout@v4
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v4
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-    - run: npm ci
-    - run: npm run build --if-present
-    - run: npm test
+[![Node.js CI](https://github.com/Huynhthuongg/Claude-OpenAI/actions/workflows/node.js.yml/badge.svg)](https://github.com/Huynhthuongg/Claude-OpenAI/actions/workflows/node.js.yml)
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)